Package.swift

```diff
@@ -127,9 +127,18 @@ let package = Package(
         "Testing",
         "_Testing_CoreGraphics",
         "_Testing_Foundation",
+        "MemorySafeTestingTests",
       ],
       swiftSettings: .packageSettings
     ),
+    .target(
```

Comment:
```suggestion
    .testTarget(
```

Reply:
This was intentional because I don't wish to incur the runtime overhead of a separate test target. I'll add an explanatory comment.